src/ol/Feature.js

```diff
@@ -97,6 +97,13 @@ class Feature extends BaseObject {
      */
     this.un;
 
+    /**
+     * Name of type of feature (e.g. GML Feature element name)
+     * @type {string|undefined}
+     * @api
```

Comment:
Does this need to be part of the API? If I understand you correctly, this is only for not losing information between GML reader and writer.

Reply:
> Does this need to be part of the API? If I understand you correctly, this is only for not losing information between GML reader and writer.

The information should also be available for use e.g. in generic interprtation of GetFeatureInfo-results (see below)